CRM/Utils/Request.php

```diff
@@ -145,7 +146,7 @@ public static function retrieve($name, $type, &$store = NULL, $abort = FALSE, $d
    * @return mixed
    *    The value of the variable
    */
-  public static function getValue($name, $method) {
+  protected static function getValue($name, $method) {
```

Comment:
This is not called from anywhere outside this function at the moment and would unsafe to use directly, hence setting to protected.